VMEncryption/main/Common.py

```diff
@@ -36,6 +36,8 @@ class CommonVariables:
     min_filesystem_size_support = 52428800 * 3
     #TODO for the sles 11, we should use the ext3
     default_file_system = 'ext4'
+    format_supported_file_systems = ['ext4', 'ext3', 'ext2', 'xfs', 'btrfs']
```

Comment:
does this trigger a need to update public documentation around xfs and btrfs support